Gemfile

```diff
@@ -13,7 +13,7 @@ group :test do
   gem "puppetlabs_spec_helper"
   gem "metadata-json-lint"
   gem "rspec-puppet-facts"
-  gem 'rubocop', '0.33.0'
```

Comment:
@jyaworski - I think this is why the build is failing for me in https://github.com/hunner/puppet-hiera/pull/127